src/gdswzd_c.F90

```diff
@@ -166,7 +166,7 @@ module gdswzd_c_mod
   !> @param[out] xlat (npts) dx/dlat in 1/degrees
   !> @param[out] ylon (npts) dy/dlon in 1/degrees
   !> @param[out] ylat (npts) dy/dlat in 1/degrees
-  !> @param[out] AREA (npts) area weights in m**2 (Proportional to the
+  !> @param[out] AREA (npts) area weights in m**2 (Proportional to the ???)
```

Comment:
I am not sure what the statement "Proportional to the" refers to. I would remove it.

Reply:
Actually I found it. It should be:

```
 * @param area (npts) area weights in m**2 (Proportional to the 
 * square of the map factor in the case of conformal projections.)

```